### DISCUSSION TITLE
What is control-plane certs expiration time & how to update it?

### DISCUSSION THREAD

**Discussion by CRASH-Tech** [siderolabs/sidero#1056] [Q&A] [answered]
subj

**Comment by smira** [comment, marked as answer]
Talos manages controlplane certs on the node automatically, you don't need to manage it.

Client-side `talosconfig` and `kubeconfig` secrets are updated by CAPI components automatically in the management cluster.

**Reply by CRASH-Tech** [reply]
Thank!
